composeApp/src/commonMain/kotlin/org/ooni/probe/ui/measurement/MeasurementScreen.kt

```diff
@@ -45,7 +49,22 @@ fun MeasurementScreen(
     LaunchedEffect(url) { Logger.i("URL: $url") }
 
     val webViewState = rememberWebViewState(url)
-    val webViewNavigator = rememberWebViewNavigator()
+    webViewState.webSettings.isJavaScriptEnabled = false
```

Comment:
By disabling javascript, we lose the ability to view raw measurement data.
![Screenshot_20241022_100007](https://github.com/user-attachments/assets/1d0d7f04-1517-43a2-a932-96ac7ab41018)

Reply:
@aanorbel I noticed, but those pages will eventually have a mobile friendly view, that won't have any link or interaction, according to @hellais. That's why I went with this change.

Reply:
https://github.com/ooni/explorer/issues/951 this is the issue on explorer

Reply:
Replied there with a comment 👍